### PR TITLE
Adding in Progress State

### PR DESCRIPTION
**Pull request by axelfernandez-eb**
Adding an Middle state

### CODE REVIEW

File: supplier_app/constants/email_notifications.py [hunk 1]
```diff
@@ -55,4 +55,15 @@
             'btn_url': '{}{}'.format(settings.BRITESU_BASE_URL, SUPPLIER_HOME_URL),
         },
     },
+    'taxpayer_in_progress': {
+        'subject': _('Your taxpayer is in progress now'),
+        'body': {
+            'upper_text': _(
+                'You Taxpayer is now in progress to Approve\n'
```
Comment (facosta-eb) [inline supplier_app/constants/email_notifications.py:62]: Your taxpayer is now in progress to be approved*

File: supplier_app/constants/email_notifications.py [hunk 2]
```diff
@@ -55,4 +55,15 @@
...
+            ),
+            'lower_text': _('You dont need to do anything, we contact with you soon, Thanks!'),
```
Comment (facosta-eb) [inline supplier_app/constants/email_notifications.py:64]: you don't need to do anything ELSE, we WILL contact you soon

File: supplier_app/constants/email_notifications.py (hunk 1 above)
Comment (axelfernandez-eb) [reply supplier_app/constants/email_notifications.py]: Done

File: supplier_app/constants/email_notifications.py (hunk 2 above)
Comment (axelfernandez-eb) [reply supplier_app/constants/email_notifications.py]: Done

File: templates/supplier_app/AP/in-progress-taxpayer.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<form method="POST" action="{% url 'handle-taxpayer-status' taxpayer_id=taxpayer.id %}">
+    {% csrf_token %}
+    <input type="hidden" name="action" value="{{in_progress}}">
+    <input class='btn btn-warning' type="submit" value="In Progress"/>
```
Comment (leonardoa-eb) [inline templates/supplier_app/AP/in-progress-taxpayer.html:4]: nit: use double quotes

File: templates/supplier_app/AP/in-progress-taxpayer.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+</form>
```
[the remark this answers is not in the document]
Comment (axelfernandez-eb) [reply templates/supplier_app/AP/in-progress-taxpayer.html]: Done

File: supplier_app/change_status_strategy.py [hunk 5]
```diff
@@ -15,7 +16,7 @@
     TAXPAYER_APPROVE_MESSAGE,
     TAXPAYER_DENIED_MESSAGE,
     TAXPAYER_REQUEST_CHANGE_MESSAGE,
-)
+    TAXPAYER_IN_PROGRESS_MESSAGE)
```
[the remark this answers is not in the document]
Comment (axelfernandez-eb) [reply supplier_app/change_status_strategy.py]: Moved!

File: supplier_app/change_status_strategy.py [hunk 6]
```diff
@@ -83,10 +84,25 @@ def show_message(request):
         messages.success(request, TAXPAYER_REQUEST_CHANGE_MESSAGE)
 
 
+class StrategyInProgress(StrategyStatusChange):
+
+    def send_email(taxpayer):
```
[the remark this answers is not in the document]
Comment (axelfernandez-eb) [reply supplier_app/change_status_strategy.py]: Thats a great Tip!!

File: templates/supplier_app/AP/in-progress-taxpayer.html (hunk 3 above)
Comment (axelfernandez-eb) [reply templates/supplier_app/AP/in-progress-taxpayer.html]: Adding double quotes in btn btn-warning

**Pull request merged**